index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <main>
+        <div class="nav">
+            <ul class="topnav">
+                <li><a href="home">Home</a></li>
+                <li><a href="tickets">Tickets</a></li>
+                <li><a href="schedule">Schedule</a></li>
+                <li style="float:right"><a href="about">About</a></li>
+            </ul>
+        </div>
+        <div class="title">
+            <h1 class="h1title">What will Game Designers <div class="big1">be Facing</div> while Desiging <div
+                    class="big2">Games?</div>
+            </h1>
+            <div class="strong">
+                <h2><a href="">Workshop+Talk w/ Three Famous Game Designers</a></h2>
+                <h2><a href="">Dec 20th, 10.30 AM EST (Zoom)</a></h2>
+                <h2><a href="">These events are free and open to the public.
+                        No prior game design experience required.</a></h2>
+            </div>
+
+            <h4>Game Designers craft the core mechanics of a game.
+                This means that they are responsible for deciding how games work when the player starts playing
+                them.
+                Most
+                importantly, game designers create rules.
+                These rules govern how games work.
+                In this workshop, participants will be introduced to what problems and rules the game designers will
+                be
+                facing
+                when
+                desiging games through speaks' life experience.
+            </h4>
+        </div>
+        <ul class="u-list-reset speaker-group">
+            <li class="speakers">
```

Comment:
You have a lot of extra markup in this `li` that is not necessary to create this layout. Compare instead this example which has a lot fewer `div`s and other markup:
https://glitch.com/edit/#!/pocky-remix-p3-layout?path=index.html